package.json

```diff
@@ -17,7 +18,8 @@
     }
   },
   "scripts": {
-    "test": "./node_modules/.bin/standard && ./node_modules/.bin/mocha test"
```

Comment:
Yup - I can see the value of linting an individual file. Good idea.

Can you revert npm test to the old behavior?
I like a command to run standard on everything, then run the test suite.

Also, maybe something's off on my computer but:

`./node_modules/.bin/standard test` doesn't seem to run standard against test.
I can make lint-breaking changed in my test files this implementation of  `npm test` still passes.